bakula/server/inboxer.py

```
@@ -31,9 +31,37 @@ def add_file_by_path(self, topic, file_path):
         # Move into the master inbox under the correct topic with an updated count
         destination = os.path.join(master_topic_path, str(counter))
         if os.path.exists(file_path):
-            os.rename(file_path, destination)
+            try:
+                os.rename(file_path, destination)
+            except:
```

Rather than a bare except statement, why not do

```python
except Exception as e:
```

Then you can just get ```e.message```.